task3/update/index.html

```diff
@@ -0,0 +1,336 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="icon" href="logo.png">
+    <title> Ribbitter | Feed </title>
+</head>
+<body>
+<header>
+    <nav class="container">
+        <a class="logo" href="index.html">
+            <img src="logo.png" alt="A VERY MUCH GREEN FROG">
+            <span class="logotext"><h1>Ribbitter</h1></span>
+        </a>
+        <a class="write" href="">
+            <img src="edit.png" alt="Write new post">
+        </a>
+        <a class="exit" href="">
+            <img src="exit.png" alt="exit">
+        </a>
+        <a class="user" href="">
+            <span class="yourname">YourName</span>
+            <img src="you.png" alt="your photo">
+        </a>
+    </nav>
+</header>
+<aside>
+    <h2 class="prefs">Your content preferences:</h2>
+    <form id="filterform">
+    <div class="filter">
+        <input type="checkbox"><span>Sort by name</span>
```

Comment:
The checkbox experience is usually provided by the following structure:
`<label><input type="checkbox"/> Checkbox Title</label>`
here the input is an actionable element that gives us functional part of experience
label - is a special element that gives us first of all visual experience, the main feature of label element is that it completely links itself with the target actionable element (that can be inside or be referenced in the "for" attribute), that means any user action on the label will be redirected to the target
e.g. in the structure above if the user clicks on text - the input will handle it.